runners/direct-java/src/main/java/org/apache/beam/runners/direct/portable/GroupAlsoByWindowEvaluatorFactory.java

```diff
@@ -138,30 +126,30 @@ public GroupAlsoByWindowEvaluator(
           .getExecutionContext(application, inputBundle.getKey())
           .getStepContext(
               evaluationContext.getStepName(application));
+      // TODO: extract from input PCollection via the graph, including length-prefixing
       windowingStrategy =
-          (WindowingStrategy<?, BoundedWindow>)
-              application.getTransform().getInputWindowingStrategy();
+          null;
 
       outputBundles = new ArrayList<>();
       unprocessedElements = ImmutableList.builder();
 
+      // TODO: extract from input PCollection via the graph, including length-prefixing
```

Comment:
I would expect that these elements would have gone through the shared code with Flink to perform these conversions and wouldn't be specialized inside of the direct runner.

Reply:
Same as above - this logic will be shared, but right now we're temporarily placing null and exploding if this is ever used.